db/migrate/20190510161837_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
```

Comment:
You are not using the `Devise` gem to create the users. Check this out: https://github.com/plataformatec/devise
Remember to read the documentation from each gem you add